controllers/signer/err_pending.go

```diff
@@ -25,9 +27,13 @@ package signer
 // answer from an external service that is indicating that the request is still
 // being processed.
 //
+// RequeueAfter can be used to specify how long to wait before retrying. By default
+// we wait for 1s before retrying.
+//
 // > This error should be returned only by the Sign function.
 type PendingError struct {
-	Err error
+	Err          error
+	RequeueAfter time.Duration
```

Comment:
Maybe add some docs about the default here (1s)? Assuming I have understood this change, this allows the issuer developer to set a custom requeue after duration?